### ISSUE TITLE
ciao-cli: add disks to k8s cluster

### ISSUE THREAD

**Issue by kaccardi** [01org/ciao#715]
Allow the user to add volumes to a k8s cluster.

depends on #711

**Comment by kaccardi** [comment]
add bat tests to test this feature end to end.

**Comment by kaccardi** [comment]
we changed our k8s strategy, so this is no longer relevant.
